en/modules/admin/pages/assemblies.adoc

```diff
@@ -1,35 +1,308 @@
+== List
 
-The options available through the assemblies’ submenu are *Info, Features/ Components, Categories, Attachments, Assembly users* and *Moderations* (<<assemblies-submenu-fig>>). To configure the various options, see 2.5 (Processes).
+To configure assemblies on the Decidim platform, click on btn:[Assemblies] in the admin sidebar menu. A list
+will appear with the existing assemblies if there are any:
 
-[#new-assembly-fig]
-._New assembly_.
-image::image74.png[image]
+image:assemblies_list.png[Assemblies list]
 
-[#assemblies-submenu-fig]
-._Assemblies' submenu_.
-image::image68.png[image]
+You can filter by the ones that are:
 
-As with processes, several types of role can be configured for users. To register an assembly's users, click on *Assembly users* from the assemblies’ submenu and click on *NEW*.
+* Published / Unpublished
+* Public / Private
 
-[#new-assembly-user-fig]
-._New assembly user_.
-image::image8.png[image]
+image:assemblies_list_filter.png[Assemblies list filtered]
 
-You need to fill in the following fields: *Name, Email* and *Role* (select from the *Administrator, Collaborator* or *Moderator* drop-down menu). Immediately afterwards click on *Create* (<<new-assembly-user-fig>>)
+You can also search by title and control how many elements are in the list.
+
+You have 4 possible actions in this list after an Assembly is created:
+
+. Export: send by email the configuration for a given assembly. Can be imported in other Decidim installation.
+. Duplicate: to duplicate this assembly.
+. Configure: to edit the metadata and configuration for a assembly.
+. Assemblies: to manage all the children assemblies for a assembly.
+. Preview: how it will look once published.
+
+== New assembly form
+
+image:assemblies_new_form.png[New assembly form]
+
+.New assembly form: General information
+|===
+|Field |Type |Description
+
+|Title
+|Required
+|Title of the assembly. For instance, "General Committee".
+
+|Subtitle
+|Required
+|Subtitle. For instance, "Let's decide our priorities together"
+
+|Order position
+|Required
+|Which order will it have in relation with the other assemblies. Having a lower number means a higher priority. For instance,
+if you have two assemblies, "Assembly a" with weight "10" and "Assembly b" with weight "0", then "Assembly b" will be first when ordering.
+
+|URL slug
+|Required
+|URL slugs are used to generate the URLs that point to this assembly. Only accepts letters, numbers and dashes, and must
+start with a letter. Example: if your domain is 'example.org' and your slug 'general-committee' then your URL will
+be https://example.org/assemblies/general-committee
+
+|Hashtag
+|Optional
+|Hashtag for Twitter
+
+|Short description
+|Required
+|A short explanation of what's your participatory process about.
+
+|Description
+|Required
+|A long explanation of what's your assembly about.
+
+|Purpose of action
+|Optional
+|Internal field that doesn't get shown publicly. To be removed in a future version.
```

Comment:
When I was writing these docs I found that we have fields that are not used. Should we remove them? For me they only bring noise.